Cargo.lock

```diff
@@ -5,10 +5,10 @@ name = "andrew"
 version = "0.2.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
- "bitflags 1.0.4 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
Accidental `cargo update`?

Reply:
It was on purpose.
Wouldn't want people using the unfixed gfx-hal version.

Reply:
could you revert this change and then update *only* the dependency you need?
`cargo update -p gfx-backend-vulkan`